### PR TITLE
@evanbacon/docs/fix router docs

### PR DESCRIPTION
**Pull request by EvanBacon**
# Why

# How

# Test Plan

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by Simek — APPROVED** [review]
👌

File: docs/pages/router/migrate/from-react-navigation.mdx [hunk 1]
```diff
@@ -108,7 +108,7 @@ function App() {
       /* @info Delete the linking configuration, this is managed by Expo Router. */
       linking={
         {
-          /* @hidden ... */ /* @end */
+          // ...
```
Comment (amandeepmittal) [inline docs/pages/router/migrate/from-react-navigation.mdx:111]:
```suggestion
          /* @hide ... */ /* @end */
```

We often `@hide` annotation to describe descriptive code length in a snippet. There was a typo with the previous one (`@hidden` could've been `@hide`).

Comment (EvanBacon) [reply docs/pages/router/migrate/from-react-navigation.mdx:111]: I did this but then the copy action was brining over a `...` which is invalid JS

Comment (amandeepmittal) [reply docs/pages/router/migrate/from-react-navigation.mdx:111]: Interesting. I will look into it. Thank you for the input.

**Pull request merged**